### PR TITLE
Added `avn ticket` content

### PR DESCRIPTION
**Pull request by ftisiot**
# What changed, and why it matters

Added the `avn ticket` content. In order to test it you need to be included in a project with support included. Contact me to know more

### CODE REVIEW

**Comment by ftisiot** [comment]
Fixes #134

File: docs/tools/cli/ticket.rst [hunk 1]
```diff
@@ -0,0 +1,78 @@
+Command reference: ``avn ticket``
+==================================
+
+Here you'll find the full list of commands for ``avn ticker``.
```
Comment (anelook) [inline docs/tools/cli/ticket.rst:4]: typo - "ticket"

File: docs/tools/cli/ticket.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+
+Create and manage support tickets
+------------------------------------
+
+Commands for managing Aiven support tickets.
```
Comment (anelook) [inline docs/tools/cli/ticket.rst:10]: Should we add "To be able to manage support tickets your project should be covered by a support contract" 

I took "covered by a support contract" from the error message, though maybe there is a better definition of what project should be for the functionality to be supported.

**Review by anelook — CHANGES_REQUESTED** [review]
Spotted a typo, and a small improvement suggestion

File: docs/tools/cli/ticket.rst (hunk 1 above)
Comment (ftisiot) [reply docs/tools/cli/ticket.rst:4]: fixed thanks!

File: docs/tools/cli/ticket.rst (hunk 2 above)
Comment (ftisiot) [reply docs/tools/cli/ticket.rst:10]: Added as a warning for now, I couldn't find any relevant material to link to. something for the future

File: docs/tools/cli/service/acl.rst [hunk 3]
```diff
@@ -29,7 +29,7 @@ Adds an Aiven for Apache Kafka ACL entry.
   * - ``--username``
     - The username pattern: accepts ``*`` and ``?`` as wildcard characters
 
-**Example:** Add an ACLs for users with username ending with ``userA`` to ``readwrite`` on topics having name starting with ``topic2020`` in the service``kafka-doc``.
+**Example:** Add an ACLs for users with username ending with ``userA`` to ``readwrite`` on topics having name starting with ``topic2020`` in the service ``kafka-doc``.
```
Comment (anelook) [inline docs/tools/cli/service/acl.rst:32]: Hmm, I wonder if this commit was not intended to this PR. But since it is just a space, I'll merge together.

**Review by anelook — APPROVED** [review]

**Pull request merged**